src/lib/components/Footers.svelte

```diff
@@ -0,0 +1,8 @@
+<script>
+    import CenterFooter from '$lib/components/CenterFooter.svelte';
+    import SidesFooter from '$lib/components/SidesFooter.svelte';
+</script>
+
+<SidesFooter />
+<br />
+<CenterFooter />
```

Comment:
I think it's worth considering that we use flexbox here rather than an explicit `<br />`. And also, we should probably just inline these two footers rather than putting them as dedicated components. That would make the flexbox styling easier.